server/index.js

```diff
@@ -6,9 +6,9 @@ var db = require('../database/db.js');
 
 
 var app = express();
-app.use('/:id', express.static(__dirname + '/../client/dist'));
+app.use('/rooms/:id', express.static(__dirname + '/../client/dist'));
```

Comment:
This is the one that did need to change, since you want to serve the index.html when you hit /rooms/:id 👍
But you can change your data fetching routes back if you'd like. Or not!

Reply:
Ok got it. Thanks for pointing this out. I'll just keep it with the /rooms/ in there :)